tests/authentication.py

```diff
@@ -16,51 +13,41 @@
 class TestTokenEndpoint(unittest.TestCase):
     @classmethod
     def setUpClass(cls) -> None:
-        config = Config()
+        cls.config = Config()
         cls.cm = CrudMethods()
```

Comment:
tutaj zależy troche od preferencji, ale zamiast klasy, w pyteście mógłbś ten `cm` i `config` zrobić jako fixtury pytestowe i przekazywać je jako argumenty do funkcji
https://docs.pytest.org/en/latest/how-to/fixtures.html

wtedy nie musiałbyś dodawać klasy, ale jeżeli dalej chciałbyś klasę (np. żeby pogrupować testy) to nie musiałbyś pisać explicit setupu. ;)

Reply:
done, dodalem config i inne rzeczy jako fixtury